libcloud/compute/drivers/digitalocean.py

```diff
@@ -93,8 +94,21 @@ class DigitalOcean_v2_NodeDriver(DigitalOcean_v2_BaseDriver, DigitalOceanNodeDri
 
     EX_CREATE_ATTRIBUTES = ["backups", "ipv6", "private_networking", "tags", "ssh_keys"]
 
-    def list_images(self):
-        data = self._paginated_request("/v2/images", "images")
+    def list_images(self, ex_private: Optional[bool] = None, ex_tag_name: Optional[str] = None):
+        """
+        List all the available images.
+
+        :param ex_private: Filters only user (private) images.
```

Comment:
Would be good to clarify that ``None`` (default value) means that no filtering is performed.

Also, is there a distinction between ``None`` and ``False``? If there isn't, I think we should just default to ``False`` and remove ``Optional`` annotation.